grid2op/Reward/_alertTrustScore.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Reward import AlertReward
+from grid2op.dtypes import dt_float
+
+SURVIVOR_TIMESTEPS = 12 #nb timesteps to be considered survivor of an attack
```

Comment:
no global variables please. This is customized in the parameters of the environment.

Reply:
I agree, it was there, I will remove it